README.md

```diff
@@ -17,3 +21,14 @@ root = T:Collection {
     }
 }
 ```
+
+The following methods are available:
+
+* **set(key, value):** Sets a value by key
+* **get(key):** Returns a value by key
+* **getset(key, value):** Atomically sets key to value and returns the old value stored at key.
```

Comment:
Can't we make `set` to behave this way? Why create an additional method for it?
In any case `getset` is a strange name...